api/models/Tag.js

```diff
@@ -53,6 +53,12 @@ const addToCommunity = (community, tag, user_id, trx) => {
       user_id: user_id,
       created_at: new Date()
     }).save({}, {transacting: trx})
+    .then(() => new TagFollow({
```

Comment:
makes sense to me -- this gives us the option to allow tag creators to unfollow straightforwardly as well